main.py

```diff
@@ -250,9 +253,14 @@ def post(self):
         db.put(user_obj)
         row_id = str(user_obj.key().id())
         logging.info("row id = " + row_id)
-        self.send_confirmation_email(email, row_id)
-        self.response.write(json.dumps(REGISTER_SUCCESS))
 
+        try:
+            self.send_confirmation_email(email, row_id)
+        except OverQuotaError:
+            self.response.write(json.dumps(ERR_OVER_EMAIL_QUOTA))
```

Comment:
Let's hold off on this for now. Thinking about this again, I'm not sure we want to do this